### DISCUSSION TITLE
We should consider pulling multiple apps with the APP_LABEL

### DISCUSSION THREAD

**Discussion by mpryc** [dora-metrics/pelorus#831] [Ideas]
Currently Pelorus tracks one app per label for all, but github exporter in a format:

```
app.kubernetes.io/name=<app_name>
```

We should consider making same labeling mechanism across all exporters, either going with:

```
app.kubernetes.io/name=<app_name>,<app2_name>,<app3_name>
```

Or documenting and changing github to support having same labels with different values per on item:
```
app.kubernetes.io/name=<app_name>
app.kubernetes.io/name=<app2_name>
app.kubernetes.io/name=<app3_name>
```

**Comment by KevinMGranger** [comment]
We need to make sure this won't mess with the calculations as they are.
